lib/extensions/active_record_extension.rb

```diff
@@ -4,14 +4,13 @@ def swap_error_key(from, to)
   end
 
   def destroy_on_blank(attributes, *keys)
-    attributes.each do |attribute|
-      item = attribute[1]
+    attributes.each do |_k, item|
       item[:_destroy] = true if keys.all? { |key| item[key].blank? }
     end
   end
 
   def truncate_value(field, length_limit)
-    self.send("#{field}=", self.send(field)&.truncate(length_limit))
```

Comment:
@peggles2 , the `self` should be used in the first case, because it's "calling a self write accessor": 
```ruby
self.send("#{field}=", send(field)&.truncate(length_limit))
```
https://github.com/rubocop-hq/ruby-style-guide#no-self-unless-required

If Rubocop was complaining, I'm guessing it's because it was tricked by `send`ing a private method rather than writing it directly, i.e. `self.whatever_field =`

Reply:
hmm yeah rubocop was complaining

Reply:
<img width="723" alt="Screen Shot 2019-08-09 at 3 41 37 PM" src="https://user-images.githubusercontent.com/7727136/62804767-38bc6700-babc-11e9-88c9-ee99b08394d5.png">